woocommerce/woocommerce
# [k6 tests] Manage K6 version

### Changes proposed in this Pull Request:

Closes: https://github.com/woocommerce/woocommerce/issues/39448

Added a script that manages K6 installation depending on OS.
Pinned the K6 version in the install script.
Updated the version to v0.53.0, currently the latest.
Updated the node scripts to handle the changes. K6 will always be installed in the `bin` folder and executed from there.

### How to test the changes in this Pull Request:

Run the node scripts locally and make sure everything works.

Install K6

```
pnpm test:perf:install-k6
```

Start perf env

```
pnpm env:perf
```

Run perf tests

```
pnpm test:perf
```

Check CI. K6 tests job should be green. Check the logs for the installed version. It should say v0.53.0. See [this run#step:5:32](https://github.com/woocommerce/woocommerce/actions/runs/10778014738/job/29893415086?pr=51239#step:5:32).
